src/pages/settings/Profile/LoungeAccessPage.js

```diff
@@ -35,18 +57,46 @@ const menuItems = [
     },
 ];
 
-function LoungeAccessPage({user}) {
+function LoungeAccessPage(props) {
```

Comment:
NAB for now but let's generally try to stick with using props destructuring. This will be required when we migrate to TypeScript